apps/src/templates/currentUserRedux.js

```diff
@@ -80,13 +81,16 @@ export const setProgressTableV2ClosedBeta = progressTableV2ClosedBeta => ({
 
 const initialState = {
   userId: null,
+  uuid: null,
   userName: null,
   userType: 'unknown',
   userRoleInCourse: CourseRoles.Unknown,
   signInState: SignInState.Unknown,
   hasSeenStandardsReportInfo: false,
   isBackgroundMusicMuted: false,
   isSortedByFamilyName: false,
+  isLti: undefined,
+  isTeacher: undefined,
```

Comment:
If we already have `userType`, do we also need isTeacher?

Reply:
Since we use this frequently, I thought it would be helpful to have it in a separate property for reusability.
But I can use `userType` instead as we usually do.

Reply:
Not a huge deal, my 2 cents is to leverage what's already there rather than introducing another prop though.